extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_iframe_transport.html

```diff
@@ -41,6 +69,38 @@
         }
       </script>
     </amp-analytics>
+    <amp-ad-exit id='exit-api'>
```

Comment:
Inclusion of amp-ad-exit isn't really necessary here, but will save me a merge conflict with the other PR I'm currently working on.